lib-es5/uploader.js

```diff
@@ -463,14 +467,12 @@ function call_api(action, callback, options, get_params) {
   var boundary = utils.random_public_id();
   var errorRaised = false;
   var handle_response = function handle_response(res) {
-    // let buffer;
     if (errorRaised) {
 
       // Already reported
     } else if (res.error) {
       errorRaised = true;
-      deferred.reject(res);
-      callback(res);
+      handleUploadAPIError(callback, res, deferred, res);
```

Comment:
`res.error`?

Reply:
No.. look at how the original error is sent